S21/hvkone/ui_txt_conv.py

```diff
@@ -0,0 +1,81 @@
+#---------------------------------
+# DEPENDECIES
+#---------------------------------
+# pip install langdetect
+from langdetect import detect
+
+# pip install translate
+from translate import Translator
+
+# pip install pycountr
```

Comment:
It nice that you have explained about the libraries you installed, however; for this code, we think this is something more and time-consuming, it can be done even without installing these libraries. 
Also, it was discussed not to use built-in libraries for this project.